INCode/diagramconfiguration.py

```diff
@@ -73,16 +96,11 @@ def revealChildren(self):
         if not current_item or current_item.childCount() > 0:
             return
         callable = current_item.get_callable()
-        if not callable.is_definition():
-            callable = self.index_.load_definition(callable)
-            if not callable.is_definition():
-                return  # TODO(KNR): really?
 
         for child in callable.get_referenced_callables():
             child.initialize()  # lazy load the referenced callables
-            CallableTreeItem(child, self.index_, current_item)
+            child_tree_item = CallableTreeItem(child, current_item)
```

Comment:
`child_tree_item` is not used. Suggest:

```suggestion
            CallableTreeItem(child, current_item)
```

Reply:
object is required below due to a recent change